src/components/Chatbot/Chatbot.tsx

```diff
@@ -23,46 +19,115 @@ const NewConversationButton = styled(PrimaryButton)(() => ({
 }));
 
 export const Chatbot = () => {
-  const [conversation, setConversation] = useState<Message[]>([]);
+  const [conversation, setConversation] = useState<MessageBubbleProps[]>([]);
   const [error, setError] = useState(false);
   const [loading, setLoading] = useState(false);
+  const [conversationCreated, setConversationCreated] = useState<string | null>(
+    null
+  );
+
+  const loadConversationFromBackend = async () => {
+    try {
+      await loadConversation().then((response) => {
```

Comment:
You are mixing `async – await` with `then`. It's better to pick one and stick with that.

Reply:
Resolved = ignored? It's causing a few bugs, so I think it was not a bad suggestion 😄